examples/angular/src/app/app.routes.ts

```diff
@@ -1,24 +1,9 @@
 import { Routes } from '@angular/router';
-
-import { canMatchPage } from './guards/can-match-page.guard';
-import { DotCMSPageResolver } from './resolver/dotcms-page.resolver';
-
 import { DotCMSPagesComponent } from './pages/pages.component';
-import { NotFoundComponent } from './pages/notFound/notFound.component';
 
 export const routes: Routes = [
   {
     path: '**',
-      resolve: {
-      // This should be called `context`.
-      context: DotCMSPageResolver,
-    },
-    canMatch: [canMatchPage],
-    component: DotCMSPagesComponent,
-    runGuardsAndResolvers: 'always', // Run the resolver on every navigation. Even if the URL hasn't changed.
-  },
-  {
-    path: '**',
-    component: NotFoundComponent
+    component: DotCMSPagesComponent
```

Comment:
@fmontes In this project is possible to have a simple page with Angular for example a path `contact` to `ContacCompoment` or all paths are handled to dotCMS?

Reply:
In this example all pages are handled by dot
The dev could add a route before the generic path or add a resolver to achieve what you say, but the example is only for pages coming from dotCMS